Autopilot/Path Manager/PathManager.c

```diff
@@ -340,7 +340,23 @@ char followWaypoints(PathData* currentWaypoint, float* position, float heading,
 
             *sp_Heading = (int)followStraightPath((float*)&waypointDirection, (float*)targetCoordinates, (float*)position, heading);
         }
-        else{
+        else{            
```

Comment:
I'm pretty sure I nested the added if statement into the else, not vice versa. This shouldn't change the current behavior. I could be wrong though.